### DISCUSSION TITLE
vcf format

### DISCUSSION THREAD

**Discussion by naio5** [gaynorr/AlphaSimR#184] [General]
Hello Dear developer,
I have used AlphasimR to simulate SNP and phenotype data for a population. SNP genotype and haplotypes were extracted using pullMarkerGeno and pullMarkerHaplo command. Does alphsim consider minor allele as reference allele? I have been trying to find a tool to convert simulated genotype data to vcf format or .ped format, but no success. Do you have any recommendations in this matter?
Best regards,
Bioen

**Comment by gregorgorjanc** [comment]
In AlphaSimR the allele 0 is ancestral and 1 a mutation from the ancestral. Hence, you will find that allele 1 tends to be less common than 0, but depending on the strength of drift and selection, allele 1 can become more common than allele 0.

There is function to write a plink file. Does that help? There is likely an R package that writes VCF files given some input. Have you explored that option? Maybe we develop an example for you and others in this discussion thread?

**Comment by naio5** [comment]
I really appreciate your reply. writePlink function worked !
